src/data/set/intervals/group.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Patrick Massot, Yury Kudryashov, Rémy Degenne
+-/
+import data.set.intervals.basic
+import algebra.order.group.basic
+
+/-! ### Lemmas about arithmetic operations and intervals. -/
```

Comment:
Those lemmas belong in `data.set.pointwise.interval` (from #17347)

Reply:
As above.

Reply:
`data.set.pointwise.interval` depends on `algebra.order.field.basic`.